.nais/dev-gcp.json

```diff
@@ -20,7 +20,12 @@
     "NARMESTELEDER_URL": "https://narmesteleder.dev.intern.nav.no",
     "NARMESTELEDER_SCOPE": "api://dev-gcp.teamsykmelding.narmesteleder/.default",
     "BASE_URL_DINE_SYKMELDTE": "https://www-gcp.dev.nav.no/arbeidsgiver/sykmeldte",
-    "SYFOMOTEBEHOV_URL": "NOT IN USE"
+    "SYFOMOTEBEHOV_URL": "NOT IN USE",
+    "DOKDIST_FORDELING_URL": "https://dokdistfordeling.dev-fss-pub.nais.io",
+    "DOKDIST_FORDELING_SCOPE": "api://dev-fss.teamdokumenthandtering.saf/.default",
+    "DOKARKIV_URL": "https://dokarkiv.dev-fss-pub.nais.io",
+    "DOKARKIV_SCOPE": "api://dev-fss.teamdokumenthandtering.dokarkiv-q1/.default",
+    "SYFOOPDFGEN_URL": "http://syfooppdfgen"
```

Comment:
Er dette riktig url? Mangler det ikke domene-navn her? Eller hvordan fungerer egentlig dette?

Det ser ut til å mangle en `P` i "SYFOOPDFGEN_URL".

Reply:
Den treffer samme namespace som poden befinner seg i (dev-gcp | team-esyfo)

Reply:
Ah, ok

Reply:
Oppfølgingsspørsmål: Kjører syfooppdfgen både i gcp og fss? Skal den kalles fra både fss og gcp?

Reply:
@kaja-nav Kan ikke se at den kjører i gcp (prod) pr. dags dato.

Reply:
Da vil det vel ikke fungere å kalle `http://syfooppdfgen` fra dev-gcp og prod-gcp når vi skal switche over?

Reply:
Nei.. ikke uten at syfooppdfgen først deployes til GCP miljøene

Reply:
Ouch. Vi bør ikke innføre noe som ikke kommer til å fungerer i gcp. Da tenker jeg at det bør løses nå når vi først er i gang med denne oppgaven.

Reply:
@Vitenok Har du noen tanker om denne?

Reply:
Har syfooppdfgen database? Hvor komplisert vil det være å flytte den til gcp?

Reply:
Nei, jeg tror det bare er kode for å generere PDF. Det som tar tid å flytte er vel konfig og GCP spesifikt oppsett for kjøring

Reply:
Koden ser veldig tynn ut, den henter det meste fra et annet image (navikt/pdfgen) og legger kun inn OP-spesfikke konfigurasjon.

Reply:
Wild shot, men er det aktuelt å generere pdf direkte i esyfovarsel i stedet for å kalle  `syfooppdfgen`?

Reply:
@kaja-nav Hvordan tenker du vi skulle gjort det esyfovarsel? Vi må vel i så fall kjøre en side container som henter pdfgen-imaget inn, og deretter kalle den via localhost? Sånn rent intuitivt tror jeg migrering av syfooppdfgen vil være enklere. Den appen er veldig tynn, og har ingen avhengigheter tror jeg?

Reply:
@torchr89 Jeg tenkte ikke, jeg bare skrev det 😄

Reply:
@kaja-nav Husker vi hadde direkte generering av pdf i syfooppfolgingsplanservice før, men så ville noen fjerne det for å legge det i egen app. Tipper det er det som er syfooppdfgen

Reply:
Mm, skjønner. Tenkte det kanskje var ganske rett frem med pdf-generering, og at man ikke nødvendigvis trenger å basere seg på navikt/pdfgen, men det er mulig det er så knotete at man bør ha det i egen app

Reply:
Kunne jo vært et bibliotek i stedet for app.

Reply:
Ja, det må jo finnes noe slikt. Mener å huske at (på den tiden) var det noen spesialbehov som ikke ble dekket av biblioteket som utviklerne hadde funnet (husker ikke konkret hva det var). Det er jo litt rart å bruke en dedikert
app til dette formålet,